pkg/network/controllers/vm_test.go

```diff
@@ -608,6 +608,56 @@ var _ = Describe("VM Network Controller", func() {
 		Expect(updatedVMI.Spec.Networks).To(Equal(originalVM.Spec.Template.Spec.Networks))
 		Expect(updatedVMI.Spec.Domain.Devices.Interfaces).To(Equal(originalVM.Spec.Template.Spec.Domain.Devices.Interfaces))
 	})
+
+	It("Could unplug an interface that is pending hotplug", func() {
```

Comment:
The name does not describe in full the scenario and it drifts from the used pattern in all other tests here.
It would be nicer to keep the format for consistency and add a bit more information.

e.g. "sync does not hotunplug interfaces when a pending hotplug without a status entry exists"